framework/rbac/DbManager.php

```diff
@@ -524,7 +524,7 @@ public function getPermissionsByRole($roleName)
      */
     public function getPermissionsByUser($userId)
     {
-        if (empty($userId)) {
+        if (!is_numeric($userId) && empty($userId)) {
```

Comment:
Pretty sure this would conflict with mongodb, since their id's are not numeric.   This PR negates the ability to utilize mongo with yii2 and utilize the O.O.B RBAC system.   I may be completely wrong in that ( it's happened before ), but pretty sure i'm not.